onnxsim/onnx_simplifier.py

```diff
@@ -111,6 +111,73 @@ def get_input_names(model: onnx.ModelProto) -> List[str]:
     return input_names
 
 
+def get_elem_size_from_elem_type(elem_type: int):
+    return np.dtype(get_np_type_from_elem_type(elem_type)).itemsize
+
+
+def get_model_info(model: onnx.ModelProto):
+    """
+    op_dict
+    weight_num
+    """
+    op_dict = dict()
+    for node in model.graph.node:
+        if node.op_type not in op_dict.keys():
+            op_dict[node.op_type] = 1
+        else:
+            op_dict[node.op_type] = op_dict[node.op_type] + 1
+    weight_num = 0
+    for init in model.graph.initializer:
+        num = 1
+        if len(init.dims) != 0:
+            for dim in init.dims:
+                num *= dim
+        else:
+            size_bytes = len(init.raw_data)
+            elem_size = get_elem_size_from_elem_type(init.data_type)
+            num = size_bytes / elem_size
+        weight_num += int(num)
+    model_info = dict()
+    model_info.update({"op_dict": op_dict})
+    model_info.update({"weight_num": weight_num})
```

Comment:
Thanks for your great contribution. Here are some of my suggestion:
1. users cares more about the byte size of model than about the "weight_num"
2. you can have a try to use `model.ByteSize()` to get the byte size